testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -4434,6 +4434,39 @@ QUnit.test("Freespace row have the correct height when using master-detail with
     this.clock.restore();
 });
 
+QUnit.test("DataGrid should apply columns that are dynamically added to a band (T815945)", function(assert) {
+    // arrange
+    this.clock = sinon.useFakeTimers();
```

Comment:
I suggest you use local variable to initialize fake timers. Also the module can already has an initialized this.clock. If so, you can use it.

Reply:
The module has not an initialized this.clock